CLANc/CLANc.munki.recipe

```diff
@@ -84,6 +88,27 @@
 				<dict>
 					<key>version</key>
 					<string>%version%</string>
+					<key>installs</key>
+					<array>
+						<dict>
+							<key>CFBundleIdentifier</key>
+							<string>%NAME%</string>
+							<key>CFBundleName</key>
+							<string>%NAME%</string>
+							<key>CFBundleShortVersionString</key>
+							<string>%version%</string>
+							<key>CFBundleVersion</key>
+							<string>For the Version number, use "About Clan..." inside the application</string>
```

Comment:
Could you add more context to that?

Reply:
I used MunkiAdmin to create the Installs Array for the Application ClanC and that was what it retrieved from the info.plist of the App. The CFBundleShortVersionString is extracted via the `Versioner` Processor.
I don't know if that is what you meant with more context.

Reply:
It doesn't look like somethin we (or more importantly: munki) could use. Should we ommit it?

Reply:
Yes I agree. We can omit that.